tmt-app/src/Components/Home.js

```diff
@@ -1,63 +1,112 @@
+                        <Typography variant="CustomHeading3">
                             No one can master multiple subjects the way you do. Spend 2 minutes 
                             with us to find the perfect major for you!
-                        </h6>
+                        </Typography>
                     </span>
-                </div>
-            </div>
+                </FlexContainer>
+            </FlexContainer>
 
-            <div className="Home_Button_Container">
-                <div className="Home_Button_Bg_Container">
+            <FlexContainer sx={{flexDirection:"row"}}>
+                <div className="Home_Button_Bg_Wrapper">
                     <div className="Home_Button_Bg_Left">
-                        <img src={LycheeRectangle} id="Home_Button_Bg_Img1" alt="LycheeRectangle"></img>
-                        <img src={MilkTeaSwiggle2} id="Home_Button_Bg_Img2" alt="MilkTeaSwiggle2"></img>
+                        <img 
+                            src={LycheeRectangle} 
+                            id="Home_Button_Bg_Img1" 
+                            alt="LycheeRectangle"
+                        />
+                        <img 
+                            src={MilkTeaSwiggle2} 
+                            id="Home_Button_Bg_Img2" 
+                            alt="MilkTeaSwiggle2"
+                        />
                     </div>
                     <div className="Home_Button_Bg_Right">
-                        <img src={ThaiTeaSwiggle1} id="Home_Button_Bg_Img3" alt="ThaiTeaSwiggle1"></img>
+                        <img 
+                            src={ThaiTeaSwiggle1} 
+                            id="Home_Button_Bg_Img3" 
+                            alt="ThaiTeaSwiggle1"
+                        />
                     </div>
                 </div>
-                <div className="Home_Button_Main">
-                    <button className="Home_Button_Main_Btn"> 
-                        Take the Quiz
-                    </button>
+                <FlexContainer 
+                    sx={{
+                        flexDirection:"column", 
+                        position:"none",
+                        height:"auto"
+                    }}
+                >
+                    <PrimaryBtn sx={{color: "#FFFFFF"}}> 
+                        <Typography variant="CustomHeading2">
+                            Take the Quiz
+                        </Typography>
+                    </PrimaryBtn>
+ 
                     <div className="Home_Button_Main_Link">
-                        <a href="google.com" target="_blank">See all Majors/Careers</a>
-                        <ArrowRightAltIcon></ArrowRightAltIcon>
+                        <Link
+                            sx={{color:"Boba.main"}}
+                            underline="none"
+                            href="google.com" 
+                            target="_blank"
+                            rel="noopener" 
```

Comment:
can we add also `noreferrer` for this line?
`noopener noreferrer` as a combo for the `rel` attribute